uiplib/gui/gallery.py

```diff
@@ -2,7 +2,9 @@
 
 from tkinter import *
 from PIL import ImageTk
+from PIL import Image
 from uiplib.uipImage import UipImage
+import os
```

Comment:
@Aniq55 yes but we do have standard imports and it'll be better if we maintain them, can you check the wikis? i think we have it there.